tools/clang/test/CodeGenDXIL/hlsl/semantics/vertex_location/sv_base_vertex_location_fal.hlsl

```diff
@@ -0,0 +1,17 @@
+// XFAIL:*
```

Comment:
This doesn't seem like a CodeGen test. Where are these errors supposed to come from? Is this validation or sema?

Reply:
Should be somewhere in sema. Moved it to SemaHLSL.